sql_generators/glean_usage/templates/baseline_clients_daily_v1.query.sql

```diff
@@ -24,7 +24,11 @@ WITH base AS (
     normalized_channel,
     normalized_os,
     normalized_os_version,
+    {% if has_distribution_id %}
+    CAST(metrics.string.metrics_distribution_id AS STRING) AS distribution_id,
```

Comment:
I don't think casting the actual column is necessary.
```suggestion
    metrics.string.metrics_distribution_id AS distribution_id,
```

Reply:
I wonder if we should keep it to make sure when the schema file is generated it is generated with correct type for all apps, WDYT @sean-rose ?

Reply:
All string metric fields should be of type `STRING`, so casting such fields as `STRING` should never be necessary.

`CAST(NULL AS STRING)` in the `else` clause is necessary because BigQuery defaults to treating raw `NULL`s as the `INTEGER` type.

Reply:
Got it, thanks for taking a moment to explain this 🙇 I made the change as requested.